sunpy/util/config.py

```diff
@@ -144,3 +148,33 @@ def _get_user_configdir():
     if not _is_writable_dir(configdir):
         raise RuntimeError(f'Could not write to SUNPY_CONFIGDIR="{configdir}"')
     return configdir
+
+
+def write_config(overwrite=False):
+    """
+    Copies the default sunpy config file to the user's config directory.
+
+    Parameters
+    ----------
+    overwrite : `bool`
+        If True, existing config file will be overwritten.
+    """
+    config_filename = 'sunpyrc'
+    module_dir = Path(sunpy.__file__).parent
+    config_file = module_dir / 'data' / config_filename
```

Comment:
If `module _dir` isn't used anywhere else (it looks like it is not?), I'd recommend doing all of this in one line
```suggestion
    config_file = Path(sunpy.__file__).parent / 'data' / config_filename
```